.ci/build-in-ci.ps1

```diff
@@ -1,4 +1,4 @@
-choco install docfx -y --version 2.48.1
+nuget.exe install docfx.console
```

Comment:
I'm quite sure you don't need the `.exe` part:

```suggestion
nuget install docfx.console
```

Reply:
True but now it appears it has failed for different reasons. Will commit though.